README.md

```diff
@@ -1 +1,40 @@
-# fluidpay-guardian
+# Guardian
+
+![release](https://github.com/fluidpay/fluidpay-guardian/actions/workflows/release.yml/badge.svg)
+
+Guardian is a user data collector tool, created for payment fraud protection.
+
+## Usage
+
+### With CDN:
+
+Add the following code snippet into your html file's head.
+
+```html
+<script src="https://unpkg.com/@gwservices/guardian@latest/dist/guardian.umd.js"></script>
+<script>
+    new Guardian('https://my_gateway_url', 'public_api_key').process();
+</script>
+```
+
+### Get the collected data
```

Comment:
Mark this section that it's optional, because to use the Guardian it's not necessary.